Tests/Rules/AvoidLongLines.tests.ps1

```diff
@@ -0,0 +1,43 @@
+$ruleName = "PSAvoidLongLines"
+
+$ruleSettings = @{
+    Enable = $true
+}
+$settings = @{
+    IncludeRules = @($ruleName)
+    Rules = @{ $ruleName = $ruleSettings }
+}
+
+Describe "AvoidLongLines" {
+    it 'Should be off by default' {
+        $def = "a" * 500
+        $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def
+        $violations.Count | Should -Be 0
+    }
+
+    it 'Should find a violation when a line is longer than 120 characters (no whitespace)' {
+        $def = "a" * 125
+        $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+        $violations.Count | Should -Be 1
```

Comment:
If there's a way to test the extent of the violating line, we should do that to make sure it makes sense.

We should probably also have a test for a multiline file/script

Reply:
I've added some multi-line script definitions to the tests, but I'm not sure about testing the extent of the violating line. When the lines are so purpose built, I'm not sure exactly what the value of doing that is. Perhaps I've misunderstood your feedback, @rjmholt?

Reply:
Basically I want us to test that PSScriptAnalyzer identifies the correct line (which I'm guessing is the line on the extent the rule gives back).

That's important for things like VSCode, so it can put the squiggly in the right place.

Reply:
Putting a test in will mean we know what's expected, and there's no off by one errors or mysterious regressions in the future.

Reply:
Ohhhh that's what you meant. Yes, I will plan to add a couple tests for that today.

Reply:
Added some assertions for getting the correct extent. And the build still passes :D